community_supplied/Kernel/kernel-tcpip/check-tcp-connection-drop-count.rule

```diff
@@ -0,0 +1,101 @@
+/*
+ * This rule checks for the TCP Embryonic connection drop count in the system.
+ * TCP Embryonic connection is a half-open connection.
+ *
+ * It shows YELLOW alarm when the count is increased by minimum 10 from its previous value.
+ * It's sampling time is 1 minute.
+ */
+iceberg {
+    topic kernel.tcpip {
+        rule check-tcp-connection-drop-count {
+            synopsis "Detect TCP Embryonic connection drops";
+            description "Check whether the TCP connection are getting dropped";
+            sensor conn-drop-sensor {
+                synopsis "Sensor for TCP Embryonic connection drops";
+                description "Returns the TCP embryonic connection drop count";
+                open-config {
+                    sensor-name /junos/kernel/tcpip/tcp;
+                    frequency 1m;
+                }
+            }
+            field tcp-conndrops-cnt {
+                sensor conn-drop-sensor {
+                    path /junos/kernel/tcpip/tcp/tcp-conndrops-cnt;
+                }
+                type integer;
+                description "Number of times the TCP connection got dropped";
+            }
+            trigger report-tcp-connection-drop-count {
+                synopsis "Trigger for TCP Embryonic connection drops";
+                description "Checks for TCP Embryonic connection drops and raises an alarm";
+                frequency 1m;
+                term no-tcp-connection-drops {
+                    when {
+                        not-equal-to "$tcp-conndrops-cnt" 0;
+                        increasing-at-most-by-value "$tcp-conndrops-cnt" {
+                            value 9;
+                            time-range 140s;
+                        }
+                    }
+                    then {
+                        status {
+                            color green;
```

Comment:
Same as above. Please address

Reply:
Color green is expected here. As part of PR 1460451 fix added "term no-tcp-connection-drops".